### PR TITLE
docs: Add Django

### PR DESCRIPTION
**Pull request by bensonyang0217**
Add Django docs, give you check

### CODE REVIEW

**Comment by bensonyang0217** [comment]
ok 修正一版

File: pages/guides/python/django.zh-TW.mdx [hunk 1]
```diff
@@ -5,4 +5,59 @@ ogImageSubtitle: 將你的 Django 專案部署到 Zeabur
...
+
+![deploy](/framework/deploy.png)
+
+部署成功後啟動網站成功畫面。
+
+![django-web](/framework/django/django.png)
+
+請注意若發生錯誤時請確認專案內是否有配置：
+
+- 檢查是否有`requirements.txt`，此文件主要用途是列出項目所需的所有 Python 套件及其版本，
```
Comment (yuaanlin) [inline pages/guides/python/django.zh-TW.mdx]:
```suggestion
- 檢查是否有`requirements.txt`，此檔案的主要用途是列出項目所需的所有 Python 套件及其版本，
```

File: pages/guides/python/django.zh-TW.mdx [hunk 2]
```diff
@@ -5,4 +5,59 @@ ogImageSubtitle: 將你的 Django 專案部署到 Zeabur
...
+  它用於確保項目在不同開發環境和生產環境中具有相同的依賴項和版本，若未配置請至專案內使用以下命令，並重新推送到 Github 上。
+
+```bash
+pip freeze > requirements.txt
+```
+
+- 檢查`settings.py`裡的`ALLOWED_HOSTS`是否有配置 ，此屬性是一個 Python List，其中包含可以訪問到 Django 應用程序的主機名稱或 IP 地址 ，可使用`ALLOWED_HOSTS = ['*']`，代表允許所有主機進行訪問，以便於測試。
```
Comment (yuaanlin) [inline pages/guides/python/django.zh-TW.mdx:63]:
```suggestion
- 檢查 `settings.py` 裡的 `ALLOWED_HOSTS` 是否有設定 ，此屬性是一個 Python List，其中包含可以訪問到 Django 應用程式的域名 ，可使用 `ALLOWED_HOSTS = ['*']`，代表允許所有域名進行訪問，以便於測試。
```

**Comment by yuaanlin** [comment]
https://docs.zeabur.com/zh-TW/guides/python/django

**Pull request merged**